frontend/lib/utils/transaction-utils.ts

```
@@ -0,0 +1,195 @@
+
+// Generate display information for transactions
+export function getTransactionDisplayInfo(transaction: UnifiedTransaction): TransactionDisplayInfo {
+  let title = ''
+  let subtitle = ''
+  let amount = ''
+  let recipient = ''
+  let token = ''
+
+  switch (transaction.transactionType) {
+    case TransactionType.TOKEN_SEND:
+      const tokenData = transaction.data as TokenTransactionData
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)